libs/stats/api/src/lib/stats.resolver.ts

```diff
@@ -0,0 +1,15 @@
+import {Query, Resolver} from '@nestjs/graphql'
+import {Stats} from './stats.model'
+import {StatsService} from './stats.service'
+
+@Resolver(() => Stats)
+export class StatsResolver {
```

Comment:
Do we maybe want to make all stats loaded independently from each other?
Even if I would for example just need the article count, the rest is still loaded. We could use resolvers here instead (search for `@Resolver`)

Reply:
are you thinking about making separate methods in service that would return each value (e.g. getArticleCount, getAuthorCount) or continue using a single method (getStats), but add parameters representing values?

Reply:
also, is there a point to do it now as we will only be using this api in a single bajour component?

Reply:
Would be 3 separate methods and would also fix the issue of them running after each other instead of the same time.

I think it would serve as a good base, but there's not much of a *need* right now. But when we want to extend this, every added stat would make the endpoint slower etc.